### PR TITLE
Various package updates

### PR DESCRIPTION
**Pull request by faulesocke**

### CODE REVIEW

File: srcpkgs/bCNC/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'bCNC'
 pkgname=bCNC
-version=0.9.14.306
-revision=2
+version=0.9.14.307
+revision=1
 archs=noarch
```
Comment (ericonr) [inline srcpkgs/bCNC/template:5]: Remove noarch.

File: srcpkgs/perl-Mojolicious/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'perl-Mojolicious'
 pkgname=perl-Mojolicious
-version=8.42
+version=8.59
 revision=1
 archs=noarch
```
Comment (ericonr) [inline srcpkgs/perl-Mojolicious/template:5]: Same

File: srcpkgs/renderdoc/template [hunk 3]
```diff
@@ -15,7 +14,7 @@ maintainer="Urs Schulz <voidpkgs@ursschulz.de>"
 license="MIT"
 homepage="https://github.com/baldurk/renderdoc"
 distfiles="https://github.com/baldurk/${pkgname}/archive/v${version}.tar.gz"
-checksum=9a78951f386fb7c3ecf7df6919d300db59719d14f0621438963ae30a75d33677
+checksum=6200099987c7cfc52360d24845742883dcfe6cb31b71c6f85372a543a9608486
 patch_args="-p1"
```
Comment (ericonr) [inline srcpkgs/renderdoc/template:18]: Shouldn't be necessary anymore.

**Review by ericonr — COMMENTED** [review]
Hi! Could you split these commits into different PRs? Since they are unrelated packages, it doesn't make sense to put them in the same one.

**Comment by faulesocke** [comment]
Okay, splitted these into #25641 #25642 #25643

**Pull request closed without merging**